p2p/node.go

```diff
@@ -425,12 +371,6 @@ func (n *QriNode) disconnected(_ net.Network, conn net.Conn) {
 	n.qis.HandleQriPeerDisconnect(pi.ID)
 }
 
-// QriStreamHandler is the handler we register with the multistream muxer
-func (n *QriNode) QriStreamHandler(s net.Stream) {
```

Comment:
This func and the below `handleStream` have been combined into the `depQriStreamHandler` func